src/v2/Components/Search/SearchBar.tsx

```diff
@@ -28,10 +26,11 @@ import { useDidMount } from "v2/Utils/Hooks/useDidMount"
 import createLogger from "v2/Utils/logger"
 import { Media } from "v2/Utils/Responsive"
 import { SearchInputContainer } from "./SearchInputContainer"
+import { withTranslation, WithTranslation } from "react-i18next"
 
 const logger = createLogger("Components/Search/SearchBar")
 
-export interface Props extends SystemContextProps {
+export interface Props extends SystemContextProps, WithTranslation {
```

Comment:
why would we do this here this way, and not with the same `useTranslation` hook?

Reply:
oh i see.. this is a class component.

even though it makes sense, I would much prefer it if we used something like https://github.com/artsy/eigen/blob/26a98c408c223efefbb0b8309448c3bb79256ab3/src/app/Scenes/Search/components/CityGuideCTA.tsx#L12-L13 (docs here at the bottom https://github.com/artsy/eigen/blob/b4484db7fb7ddfbad4e903c993628bbd90a909b3/docs/palette_v3_migration_guide.md)
so we can keep everything nice and centralized, with a `t` exposed.

Reply:
I completely agree that your suggestion would be much cleaner, if it's possible to open a PR on top of this one, that would be great. I just don't want to put any pressure on you either, I'm already super grateful for the careful code review. ❤️

Reply:
I'll take a quick stab at it.